src/lgtm/scripts/download/loki.sh

```
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euo pipefail
+source "$(dirname "$(dirname "$(readlink -f "$0")")")/lib/download.sh"
+
+# Prepare
+TEMP_DIR="$(mktemp -d)"
+trap "rm -rf '$TEMP_DIR'" EXIT
```

Shellcheck (0.10.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)